.github/workflows/ci_test-full.yml

```diff
@@ -62,8 +62,13 @@ jobs:
 
     - name: Install dependencies
       run: |
-        # python -m pip install --upgrade --user pip
-        pip install --requirement requirements/devel.txt --pre --upgrade --quiet --find-links https://download.pytorch.org/whl/cpu/torch_stable.html
+        python -m pip install --upgrade --user pip
+
+        # Install only mandatory dependencies and check if `import pl_bolts` is successful without any optional packages.
+        pip install --pre --upgrade --quiet --find-links https://download.pytorch.org/whl/cpu/torch_stable.html .
+        python -c 'import pl_bolts'
```

Comment:
this shall fail, right?

Reply:
No, it shouldn't if we handle the optional imports correctly.